MacrobenchmarkSample/app/proguard-rules.pro

```diff
@@ -18,4 +18,11 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+
+# Turn off obfuscations for Baseline Profile Generation
+# This is because Baseline Profile rules participate in the build and will eventually get
+# obfuscated with the same obfuscation map that is being used for the APK. We don't want the
+# rules that are generated by BaselineProfileRule to be obfuscated on their own. We want the APK and
+# the profile rules to be obfuscated together.
+-dontobfuscate
```

Comment:
can we have this as a separate `do-not-obfuscate.pro` that's only used in the benchmark build type? that will match what external devs would do more closely (hopefully we don't need a separate variant for that)

Reply:
Agree with separate proguard file .. maybe `proguard-rules-benchmark.pro` not to confuse with the name?